source/blog/2017-08-21-ember-2-15-released.md

```diff
@@ -86,16 +73,31 @@ export default Ember.Component.extend({
 });
 ```
 
+**Phase 2** is pending implementation of the new public `RouteInfo` API. It is
+not included in Ember 2.15.0.
+
+* Expand the service with the methods `isActive`, `recognize`, and
+  `recognizeAndLoad`.
+* Deprecate the `willTransition` and `didTransition` router hooks (not the
+  route actions of the same name). Replace them with events emitted by the
+  router service which have improved timing and public API arguments:
+  `routeWillChange` and `routeDidChange`.
+
+Addons that wish to use the router service API and support Ember releases prior to
+2.15.0 should consider the
+[ember-router-service-polyfill](https://github.com/rwjblue/ember-router-service-polyfill).
+
+
 #### `{{mount}}` helper `model` argument
 
-Ember 2.15 implements
+Ember 2.15.0 implements
 [RFC #225](https://github.com/status200/rfcs/blob/ember-engines-mount-params/text/0000-ember-engines-mount-params.md) with some minor tweaks. The `{{mount}}`
 helper, used to invoke an Ember engine, now accepts the named argument of
-`model` to provide the model of the engine's application controller.
+`model` for an engine's application controller.
 
 For example, in an application you might pass some values from a template:
 
-```hbs
+```handlebars
```

Comment:
No. It is an example of a template in an application, there is no specific location on disk that is relevant.